src/teleop/gui/src/components/FlightAttitudeIndicator.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
Apparently the (the FlightAttitude Indicator) doesn't update the bearing on all GUIs except for the Auton GUI (despite it being in TF tree). Also the visual does not really update on any GUI